test/gtest/ucm/malloc_hook.cc

```diff
@@ -301,7 +301,7 @@ class test_thread {
 private:
     typedef std::pair<void*, void*> range;
 
-    bool is_ptr_in_range(void *ptr, size_t size, const std::vector<range> &ranges) {
+    bool __attribute__((noinline)) is_ptr_in_range(void *ptr, size_t size, const std::vector<range> &ranges) {
```

Comment:
can use UCS_F_NOINLINE

Reply:
done

Reply:
Actually, I'm not sure we have to make this change, checking

Reply:
No need for this commit since https://github.com/openucx/ucx/pull/10131/commits/420caff458f0e3b11fcdb2052c179980fc9d5fdf